include/edm4hep/Constants.h

```diff
@@ -21,12 +21,33 @@
 
 #include <cstdint>
 
+#define DEPRECATED_LABEL(name, newname)                                                                                \
+  static constexpr const auto name [[deprecated("Use 'edm4hep::labels::" #newname "' instead")]] = labels::newname
+
 namespace edm4hep {
-static constexpr const char* CellIDEncoding = "CellIDEncoding";
-static constexpr const char* EventHeaderName = "EventHeader";
-static constexpr const char* EventWeights = "EventWeightNames";
-static constexpr const char* shapeParameterNames = "shapeParameterNames";
-static constexpr const char* EventFilterStats = "EventFilterStats";
+namespace labels {
+  static constexpr const char* CellIDEncoding = "CellIDEncoding";
+  static constexpr const char* EventHeader = "EventHeader";
+  static constexpr const char* EventWeights = "EventWeightNames";
```

Comment:
```suggestion
  static constexpr const char* EventWeightsNames = "EventWeightNames";
```
I would align that with the others

Reply:
I don't think we need to include `Names` now that is namespaced and that information is already encoded in the namespace. Before they were in the global namespace and that information was only in `Names`